### ISSUE TITLE
Rule "semicolons: 'always'" doesn't work for variables.

### ISSUE THREAD

**Issue by Kamilius** [rossPatton/stylint#315]
stylint version: 1.3.10
OS: MacOS X El Capitan

.stylintrc:
```json
{
  "borderNone": false,
  "blocks": "always",
  "brackets": "always",
  "colons": "always",
  "colors": false,
  "commaSpace": "always",
  "commentSpace": false,
  "cssLiteral": "never",
  "depthLimit": false,
  "duplicates": true,
  "efficient": "always",
  "exclude": [],
  "extendPref": "@extend",
  "globalDupe": false,
  "indentPref": 2,
  "leadingZero": "never",
  "maxErrors": false,
  "maxWarnings": false,
  "mixed": false,
  "namingConvention": false,
  "namingConventionStrict": false,
  "none": "always",
  "noImportant": false,
  "parenSpace": "never",
  "placeholders": false,
  "prefixVarsWithDollar": "always",
  "quotePref": "single",
  "semicolons": "always",
  "sortOrder": "alphabetical",
  "stackedProperties": "never",
  "trailingWhitespace": "never",
  "universal": false,
  "valid": true,
  "zeroUnits": "never",
  "zIndexNormalize": false
}

```

I have a separate file with colors, in format:

```scss
$color-black = #000
$color-white = #fff;
```

When I try to lint this file, linter's not detecting missing semicolons.

**Comment by hikkyu** [comment]
Hi!

It seems to be the same with:

```stylus
background: $colors.bg
color: #fff
```
It breaks only on `color` property with `"semicolons": "always"`

**Comment by rossPatton** [comment]
this should be fixed in the latest version (1.5.0)
